### DISCUSSION TITLE
Audi A4 MIB & Update MAPS need help

### DISCUSSION THREAD

**Discussion by bluewebcity** [Mr-MIBonk/M.I.B._More-Incredible-Bash#150] [Q&A]
Audi A4 2016y  
Current version: MHI2_ER_AUG22_P2133
MU Software: 0358
I don't have 023000e, I have 0230001e. I wanted to update the maps. I installed the MIB, downloaded pathes, copied it to the patch folder, got the green menu, did everything as shown in the video, but I still don't have 023000e, on the green menu it says There is no Flashfile. Can you help me figure out where I went wrong, as soon as possible? Thanks in advance.

**Comment by harman-f** [comment]
LOG file!

**Reply by bluewebcity** [reply]
[MU0358-JTB-07411.12.151176031B.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11008953/MU0358-JTB-07411.12.151176031B.log)
![IMG-6277986f37a8f11a9316e2454c82603b-V](https://user-images.githubusercontent.com/64874210/226117019-9261ca74-eef6-4584-bc76-49c1e72f196d.jpg)
![IMG-cc88fef2539b8f457a59f5897f5bf08b-V](https://user-images.githubusercontent.com/64874210/226117028-a86eb242-6c62-44ee-ad54-813afdb50d12.jpg)

**Reply by harman-f** [reply]
Very old FW. 
Update and M.I.B will work

**Reply by bluewebcity** [reply]
Thank you for your time, I will ask you something else, is this video what I need https://youtu.be/6FlHM9VZl_s (MIB2 firmware update tutorial VW Audi Skoda Seat download links) and whether on mibsolution.one i need to download firmware file MHI2_ER_AUG22_P3241_1 MU0678 (A 4-A 5- K 5- K 7). or  This is all new to me, so I hope you'll understand me for bothering you so much, thank you very much once again.
